scripts/generate.sh

```diff
@@ -13,7 +13,7 @@ ANNOPREFIX=${1:-zanzibar}
 bash ./codegen/runner/pre-steps.sh "$PREFIX/build" "$PREFIX" "$ANNOPREFIX"
 
 start=$(cat .TMP_ZANZIBAR_TIMESTAMP_FILE.txt)
-go run codegen/runner/runner.go -config="$PREFIX/build.json"
```

Comment:
This is a breaking change; you should check for `build.yaml` and fall back to `build.json`